spec/features/studios/index_spec.rb

```diff
@@ -0,0 +1,43 @@
+require "rails_helper"
+
+RSpec.describe "studio index" do
+  before :each do
+    @studio_1 = Studio.create!(name: "MAPPA Studio", location: "Suginami")
+    @studio_2 = Studio.create!(name: "UFO Table", location: "Shibuya")
+
+    @movie_1 = @studio_1.movies.create!(title: "Jujutsu Kaisen", creation_year: "2018", genre: "Adventure Dark", studio_id: @studio_1.id)
+    @movie_2 = @studio_1.movies.create!(title: "Attack on Titan", creation_year: "2013", genre: "Action Dark", studio_id: @studio_1.id)
+
+    @movie_3 = @studio_2.movies.create!(title: "Demon Slayer", creation_year: "2016", genre: "Action Fantasy", studio_id: @studio_2.id)
+    @movie_4 = @studio_2.movies.create!(title: "Fate/Zero", creation_year: "2011", genre: "Dark fantasy", studio_id: @studio_2.id)
+  end
+
+  describe "US 1- Studio Index Page" do
+    describe "When i visit the studip index page I see all the studios and their info" do
+      it "Also displays all their movies and their moveis info" do
+        visit "/studios"
+
+        expect(page).to have_content(@studio_1.name)
+        expect(page).to have_content(@studio_1.location)
+        expect(page).to have_content(@studio_2.name)
+        expect(page).to have_content(@studio_2.location)
+
+        expect(page).to have_content(@movie_1.title)
```

Comment:
In your view, you do a great job anchoring these variables to labels  (i.e. `Title:    movie.title`)
Consider making your assertions here more robust by anchoring this data to these same titles. Example:
`expect(page).to have_content("Creation Year: #{@movie_1.creation_year}")`

Reply:
This will allow you to test that this data is showing up in the right place, rather than randomly floating on the page somewhere